chainpoint/chainpoint-gateway
# Defect PT#157924036: Node UI LocalStorage Defect Resolution

### Changes

* Refactored Redux state. More specifically, moved all node related config + stats data into 'node' reducer. 'app' reducer now contains just auth data along with other app-level metadata
* Removed npm package `redux-localstorage`. We're now using `redux-persist` which supports migrations, along with other capabilities we may leverage in the future.
* Making a call to remove previous localStorage that was persisted under the 'redux' key. This call to remove an item from localStorage is idempotent so it will not cause any error to be thrown regardless of whether or not previous redux state has been persisted into localStorage

### How to test:
1. Launch Node UI
2. Open DevTools
3. Inspect localStorage data being persisted
4. Expect one key with a value of 'persist:root' to exist
5. Expect no other key/value pair to be persisted into localStorage
